lib/ansible/modules/packaging/language/maven_artifact.py

```diff
@@ -263,19 +272,22 @@ def parse(input):
 
 
 class MavenDownloader:
-    def __init__(self, module, base="http://repo1.maven.org/maven2"):
+    def __init__(self, module, base="http://repo1.maven.org/maven2", local=False):
         self.module = module
         if base.endswith("/"):
             base = base.rstrip("/")
         self.base = base
+        self.local = local
```

Comment:
In the future, if we have more than 3-4 schemes, maybe we can have a `scheme` class attribute instead, however I think it is fine by now.

Reply:
Ok for next time